05-Jason-Luo/wk04/3-wed/book search/main.rb

```diff
@@ -0,0 +1,26 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'httparty'
+# ERB: Embedded Ruby: HTML with chunks of Ruby embedded in it.
+
+
+get '/' do
+    # "hello from the home page"
+    erb :home # automatically searches views/
+end
+
+get '/info' do
+
+    # @instance variable
+    title = params[:book_title]
+    book_url = "https://www.googleapis.com/books/v1/volumes?q=title:#{ title }"
+    
+    info = HTTParty.get book_url
+    @title = info["items"][0]["volumeInfo"]["title"]
+    @author = info["items"][0]["volumeInfo"]["authors"][0]
+    @description = info["items"][0]["volumeInfo"]["description"]
+    @image = info["items"][0]["volumeInfo"]["imageLinks"]["thumbnail"]
```

Comment:
Theres some repetition here, I'd recommend handling the data like this:
```suggestion
    book_info = info["items"][0]["volumeInfo"]
    @title = book_info["title"]
    @author = book_info["authors"][0]
    @description = book_info["description"]
    @image = book_info["imageLinks"]["thumbnail"]
```